files/build_templates/sonic_debian_extension.j2

```diff
@@ -724,6 +724,13 @@ sudo LANG=C chroot $FILESYSTEM_ROOT sonic-package-manager install {{ name }}=={{
 sudo LANG=C chroot $FILESYSTEM_ROOT sonic-package-manager install --from-tarball {{ path }} {{ get_install_options(set_owner, enabled) }}
 {% endfor -%}
 
+{% if install_third_party_debs -%}
+TEMP_DIR="$(sudo LANG=C chroot $FILESYSTEM_ROOT mktemp -d)"
+sudo wget {{ install_third_party_debs.strip().replace(",", " ") }} --directory-prefix="$FILESYSTEM_ROOT/$TEMP_DIR"
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT bash -c "apt-get install -y \"$TEMP_DIR\"/*"
```

Comment:
@stepanblyschak Just a suggestion: maybe it's better to have *.deb, here just in case wget will download some garbage. What do you think?

Reply:
Input for wget is from user. Following principle "Garbage in, garbage out". So not hiding bad input.

Reply:
@stepanblyschak ok